### ISSUE TITLE
Password Reset Email is Blank

### ISSUE THREAD

**Issue by nickian** [mautic/mautic#1998]
When I initiate a password reset email, I receive a blank message in Mautic 2.0. I found this issue here as well for 1.4: https://github.com/mautic/mautic/issues/1783 

Was this never fixed, or am I experiencing a new issue?

**Comment by escopecz** [comment]
I tried to replicate, but the email wasn't blank. Here is how it looked for me: https://github.com/mautic/mautic/pull/2008

**Comment by npracht** [comment]
Duplicate https://github.com/mautic/mautic/issues/1783 ?

**Comment by nickian** [comment]
Not sure how to diagnose this. I tried doing a reset by both username and email, and I even tried changing from SES to PHP Mail to see if that made a difference. I receive a blank message either way.

**Comment by nickian** [comment]
Maybe it has something to do my PHP version? On my server I get a blank email. On localhost, running PHP 7.0, I get this, but what's with the image?:

```
Hi Nick,

You requested a password reset. Click the link below to reset your password.

http://localhost/mautic/passwordresetconfirm?token=0024d7ad67ebdebd2b09e5baa4e1dcac32e0bafd8d2d22fd318f333047e0d100<img style="display: none;" height="1" width="1" src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" alt="Mautic is open source marketing automation" />`

**Comment by escopecz** [comment]
@nickian if you look at #2008, I fixed the image thing and even explained what it is.

**Comment by MaxWebmecanik** [comment]
@nickian @escopecz What email clients are you using?
We've had some feedback that gmail does **not** display the email while some others do.

**Comment by escopecz** [comment]
I was using GMail for testing this.

**Comment by MaxWebmecanik** [comment]
@escopecz Ok then nevermind thanks :sweat_smile:

**Comment by nickian** [comment]
I was using Roundcube and iOS mail. My impression was that this had something to do with my PHP version. When I tested on PHP 7 it worked, but my production server is running PHP 5.6.23 and it does not work.

**Comment by escopecz** [comment]
I have PHP 5.6.23 and it works for me. Must be something else

**Comment by andersonid** [comment]
I'm experiencing the same issue... Is it solved?   I'm on Mautic 2.1.0, mysql database imported from previous install 2.0.1 , followed by "Uh Oh - John Tutorial" for update database schema, etc...

**Comment by alanhartless** [comment]
Hi guys. Checkout #2285
